### PR TITLE
add check Duplicate

### PR DESCRIPTION
**Pull request by kazukazuinaina**
Hello and Happy New Year!
I added checking to duplicate.
Could you check this Pull Request?

### CODE REVIEW

File: plugin/gista.vim [hunk 1]
```diff
@@ -1,3 +1,7 @@
+if exists('g:loaded_gista')
+  finish
+endif
+
```
Comment (lambdalisue) [inline plugin/gista.vim]: **must**

You need to add the following line as well

```vim
let g:loaded_gista = 1
```

**Comment by kazukazuinaina** [comment]
Ah, sorry I mistaked.
I fixed!

**Comment by lambdalisue** [comment]
thanks 👍

**Comment by kazukazuinaina** [comment]
@lambdalisue 
Thanks for checking!

**Pull request merged**